### PR TITLE
This /dev/null was extraneous and preventing push

### PR DESCRIPTION
**Pull request by rgbkrk**
Third times a charm? :wink:

Verified this one locally as well with my own token.

### CODE REVIEW

**Comment by blink1073** [comment]
Wasn't the /dev/null there to prevent information leak?  https://medium.com/@nthgergo/publishing-gh-pages-with-travis-ci-53a8270e87db

File: scripts/travis_after_success.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ then
 
     git add .
     git commit -m "Deployed to GitHub Pages"
-    git push --force --quiet "https://${GHTOKEN}@${GH_REF}" master:gh-pages /dev/null 2>&1
```
Comment (blink1073) [inline scripts/travis_after_success.sh]: I think it should be `> /dev/null`

Comment (rgbkrk) [inline scripts/travis_after_success.sh]: :astonished: oh god

**Comment by rgbkrk** [comment]
Fixed. So glad you read through that. Thanks @blink1073

File: scripts/travis_after_success.sh (hunk 1 above)
Comment (blink1073) [inline scripts/travis_after_success.sh]: Amateur :stuck_out_tongue_winking_eye:

**Comment by blink1073** [comment]
At least one of us is responsible.  :wink:

**Comment by blink1073** [comment]
It worked!  http://jupyter.org/jupyter-js-services/

**Comment by blink1073** [comment]
:beers:

**Comment by rgbkrk** [comment]
:smile: :+1: 

:beers:

**Pull request merged**